### PR TITLE
[zh] document translation sync

### PR DESCRIPTION
**Pull request by liu-xg**
## Description

doc localization

## Reviewers

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

content/en/docs/tasks/observability/metrics/telemetry-api/index.md
content/en/docs/tasks/security/cert-management/custom-ca-k8s/index.md

### CODE REVIEW

File: content/zh/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 1]
```diff
@@ -164,3 +164,32 @@ spec:
...
+
+{{< text bash >}}
+$ curl "http://$GATEWAY_URL/productpage"
+{{< /text >}}
+
+{{< tip >}}
+`$GATEWAY_URL` 值设置在实例 [Bookinfo](/zh/docs/examples/bookinfo/) 中。
+{{< /tip >}}
+
+使用以下命令验证 Istio 是否为您的新文件生成了数据或者修改过的空间维度:
```
Comment (wilsonwu) [inline content/zh/docs/tasks/observability/metrics/telemetry-api/index.md:180]:
```suggestion
使用以下命令验证 Istio 是否为您的新文件生成了数据或者修改过的空间维度：
```

File: content/zh/docs/tasks/observability/metrics/telemetry-api/index.md [hunk 2]
```diff
@@ -164,3 +164,32 @@ spec:
                 mode: SERVER
                 metric: REQUEST_COUNT
     {{< /text >}}
+
+## 验证结果 {#verify-the-results}
+
+发送流量到网格。对于 Bookinfo 示例，在 Web 浏览器中访问 `http://$GATEWAY_URL/productpage` 或发出以下命令:
```
Comment (wilsonwu) [inline content/zh/docs/tasks/observability/metrics/telemetry-api/index.md:170]:
```suggestion
发送流量到网格。对于 Bookinfo 示例，在 Web 浏览器中访问 `http://$GATEWAY_URL/productpage` 或发出以下命令：
```

**Comment by zirain** [comment]
请不要一直开关PR，可以学习下github的基本操作.
尽量在本地运行make lint成功再push

**Comment by liu-xg** [comment]
> 请不要一直开关PR，可以学习下github的基本操作. 尽量在本地运行make lint成功再push

ok

File: content/zh/docs/tasks/observability/metrics/telemetry-api/index.md (hunk 2 above)
Comment (liu-xg) [reply content/zh/docs/tasks/observability/metrics/telemetry-api/index.md]: ok

**Review by wilsonwu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**